### PR TITLE
Voyages Optimisation_type is changed ;)

### PR DESCRIPTION
**Pull request by ghost**
Currently voyages.optimisation_type is changed from **FLOAT** to **enum**
`optimisation_type` 
`enum('fixed_eta', 'earliest_arrival', 'least_cost_arrival')`

### CODE REVIEW

File: src/server/migrations/20190924002240_voyages_optimisation_type.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+exports.up = function(knex) {
+  return knex.schema.alterTable("voyages", table => {
+    table
+      .enu("optimisation_type", [
+        "fixed_eta",
+        "earliest_arrival",
+        "least_cost_arrival"
+      ])
```
Comment (zkwsk) [inline src/server/migrations/20190924002240_voyages_optimisation_type.js:8]: Looks good, but now you have to fix the seed for voyages 😉

**Review by zkwsk — CHANGES_REQUESTED** [review]

File: src/server/migrations/20190924002240_voyages_optimisation_type.js (hunk 1 above)
Comment (ghost) [reply src/server/migrations/20190924002240_voyages_optimisation_type.js:8]: I should make and push it in this branch??
Or I have to make a new branch and send it as a new PR??

File: src/server/seeds/dev/008_00_voyages.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ exports.seed = function(knex, Promise) {
           forwarddraft: 2323.22,
           aftdraft: 2242.24,
           optimisation_type: 1,
-          depart_from_waypoint: 1,
+          depart_from_waypoint: "fixed_eta",
```
Comment (zkwsk) [inline src/server/seeds/dev/008_00_voyages.js]: You edited the wrong line.

**Review by zkwsk — CHANGES_REQUESTED** [review]

File: src/server/seeds/dev/008_00_voyages.js (hunk 2 above)
Comment (ghost) [reply src/server/seeds/dev/008_00_voyages.js]: you have an eagle eye.

**Review by zkwsk — APPROVED** [review]

**Pull request merged**